<launch>
  <node pkg="ltme01_driver" type="ltme01_driver_node" name="ltme01_driver_node" output="screen">
    <!-- Device node for LTME-01B -->
    <param name="device" value="/dev/ltme01"/>

    <!-- Frame ID used by the published LaserScan messages -->
    <param name="frame_id" value="ltme01"/>

    <!-- Start and end angle of published scans -->
    <!-- As LTME-01 has an FOV of 270 degrees, the minimum allowed value for angle_min is -2.356 (about -3 * pi / 4), and the maximum allowed value for angle_max is 2.356 (about 3 * pi / 4) -->
    <!--
    <param name="angle_min" value="-1.571"/>
    <param name="angle_max" value="1.571"/>
    -->

    <!-- Range of angle for which data should be excluded from published scans -->
    <!-- Leave these two parameters commented out if a full 270 degree FOV is desired -->
    <!--
    <param name="angle_excluded_min" value="-0.785"/>
    <param name="angle_excluded_max" value="0.785"/>
    -->

    <!-- Minimum and maximum range value of published scans -->
    <!-- Defaults to 0.2 and 30 respectively if not specified -->
    <!--
    <param name="range_min" value="0.2"/>
    <param name="range_max" value="30"/>
    -->
  </node>
</launch>
